packages/flutter_tools/lib/src/base/os.dart

```diff
@@ -208,21 +208,49 @@ class _PosixUtils extends OperatingSystemUtils {
 
   @override
   void zip(Directory data, File zipFile) {
-    _processUtils.runSync(
-      <String>['zip', '-r', '-q', zipFile.path, '.'],
-      workingDirectory: data.path,
-      throwOnError: true,
-    );
+    try {
+      _processUtils.runSync(
+        <String>['zip', '-r', '-q', zipFile.path, '.'],
+        workingDirectory: data.path,
+        throwOnError: true,
```

Comment:
should we enable `verboseExceptions` here too?

Reply:
done

Reply:
wonder if that will make any tests fail...

Reply:
pretty sure every time you ask that question, the answer is yes